deploy/materialized_view_address.sql

```diff
@@ -0,0 +1,94 @@
+                physical_address_street_number varchar(1000) path './PhysicalAddress/StreetNumber[normalize-space(.)]',
+                physical_address_street_number_suffix varchar(1000) path './PhysicalAddress/StreetNumberSuffix[normalize-space(.)]',
+                physical_address_street_name varchar(1000) path './PhysicalAddress/StreetName[normalize-space(.)]',
+                physical_address_street_type varchar(1000) path './PhysicalAddress/StreetType[normalize-space(.)]',
+                physical_address_street_direction varchar(1000) path './PhysicalAddress/StreetDirection[normalize-space(.)]',
+                physical_address_prov_terr_state varchar(1000) path './PhysicalAddress/ProvTerrState[normalize-space(.)]',
+                physical_address_postal_code_zip_code varchar(1000) path './PhysicalAddress/PostalCodeZipCode[normalize-space(.)]',
+                physical_address_country varchar(1000) path './PhysicalAddress/Country[normalize-space(.)]',
+                physical_address_national_topographical_description varchar(1000) path './PhysicalAddress/NationalTopographicalDescription[normalize-space(.)]',
+
+                mailing_address_delivery_mode varchar(1000) path './MailingAddress/DeliveryMode[normalize-space(.)]',
+                mailing_address_po_box_number varchar(1000) path './MailingAddress/POBoxNumber[normalize-space(.)]',
+                mailing_address_street_number varchar(1000) path './MailingAddress/StreetNumber[normalize-space(.)]',
+                mailing_address_street_number_suffix varchar(1000) path './MailingAddress/StreetNumberSuffix[normalize-space(.)]',
+                mailing_address_street_name varchar(1000) path './MailingAddress/StreetName[normalize-space(.)]',
+                mailing_address_street_type varchar(1000) path './MailingAddress/StreetType[normalize-space(.)]',
+                mailing_address_municipality varchar(1000) path './MailingAddress/Municipality[normalize-space(.)]',
+                mailing_address_prov_terr_state varchar(1000) path './MailingAddress/ProvTerrState[normalize-space(.)]',
+                mailing_address_postal_code_zip_code varchar(1000) path './MailingAddress/PostalCodeZipCode[normalize-space(.)]',
+                mailing_address_country varchar(1000) path './MailingAddress/Country[normalize-space(.)]',
+                mailing_address_additional_information varchar(1000) path './MailingAddress/AdditionalInformation[normalize-space(.)]',
+
+                geographic_address_latitude varchar(1000) path './GeographicAddress/Latitude[normalize-space(.)]',
+                geographic_address_longitude varchar(1000) path './GeographicAddress/Longitude[normalize-space(.)]'
+         ) as address_details
+) with no data;
+
+-- THIS WILL FAIL
+-- addresses for contacts are not currently covered with this materialized view
+-- TODO: figure out what we are doing with addresses from contacts
```

Comment:
Contacts are separate from facility and organisation so this matview currently breaks on addresses related to contacts since both facility_id and organisation_id will be null. What is the plan for handling contacts? @wenzowski @hamzajaved

Reply:
The plan is now to extract both directly, right?